modules/plot/main/coffee/Axis.coffee

```diff
@@ -70,8 +70,8 @@ class Axis
       series: new hx.List
     }
 
-    @x = dimension(this, opts.x)
-    @y = dimension(this, opts.y)
+    @x = dimension this, hx.merge axisTickLabelPosition: 'bottom', options?.x
```

Comment:
Might be clearer formatted like this:

    hx.merge({
      axisTickLabelPosition: 'bottom'
    }, options.x)